governance/ExternalGovernance.md

```diff
@@ -0,0 +1,28 @@
+# External Governance
+
+## Goals:
+1) Actively vote on Kusama governance when there is clear agreement within our diverse decentralized organization of highly involved community builders. By organizing a single collective public voting account, we can demonstrate leadership within the community.
+2) Offer community members that have no time to get involved, a way to make their vote count for OpenGov while being sure it reflects what's best for the community.
```

Comment:
Comma splice (just remove it)

Reply:
removed. still an awkward sentence, but good enough for now